### DISCUSSION TITLE
[Question] How to pass a variable content from server to brwoser.

### DISCUSSION THREAD

**Discussion by educob** [quasarframework/quasar#10573] [CLI - SSR mode]
Hi.

I am in the process of replacing graphql  with a libp2p node on the browser (and on the server of course).
So I need the browser to know the libp2p address of the server (something like: /ip4/72.xx.xx.xx/tcp/14577/http/p2p-webrtc-direct/p2p/Qmby... )

How can the frontend server send this string to the browser directly? (I mean no axios thing on the browser)

Thanks.

**Comment by ontwikkelfabriek** [comment]
@educob 

> How can the frontend server send this string to the browser directly? (I mean no axios thing on the browser)

sockets? 

Did you see this vue libp2p repo:
https://github.com/filipesoccol/libp2p-pubsub-vue

**Reply by educob** [reply]
HI. thanks for answering.

I don't think this example applies to what I need.
The browser's node needs to know which is the libp2p address of a server. I'd like to send the browser this information so the libp2p browser node can connect with the libp2p server node.

Thaks.

**Reply by ontwikkelfabriek** [reply]
@educob 

Did you know about these JS specific libp2p repo:
https://github.com/libp2p/js-libp2p

And examples:
https://github.com/libp2p/js-libp2p/tree/master/examples

What kind of software are you developing with this libp2p if I may ask? ( just curious what you can do with libp2p)
